ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerProxy.java

```diff
@@ -236,8 +236,10 @@ void initGroups(RaftGroup group) {
                   addGroup(RaftGroup.valueOf(groupId));
                 }
               } catch (Exception e) {
-                LOG.warn(getId() + ": Failed to initialize the group directory "
-                    + sub.getAbsolutePath() + ".  Ignoring it", e);
+                if (LOG.isDebugEnabled()) {
+                  LOG.debug(getId() + ": Failed to initialize the group directory "
+                      + sub.getAbsolutePath() + ".  Ignoring it", e);
+                }
```

Comment:
How about printing an info message but not the stack trace?
```
                LOG.info("{}: The directory {} is not a group directory; ignoring it. {}",
                    getId(), sub.getAbsolutePath(), e);
```

Reply:
Good idea, I will update PR.

Reply:
Updated as suggested.